Project/backend/codebase/graph_creator/graph_handler.py

```diff
@@ -422,5 +438,6 @@ def connect_with_llm(data, text_chunks, rate_limit):
         )
     )
     data = add_relations_to_data(data, connecting_relations)
+    data = add_topic(data)
```

Comment:
@nikolas-rauscher this part is connected to task #102. First we need to delete the duplicates and then run `add_topic` function. Since you are implementing **102** feel free to change the call sequence.